src/feedback/entities/feedback.entity.ts

```
@@ -8,8 +9,18 @@ export class Feedback extends BaseTimeEntity {
   @PrimaryGeneratedColumn()
   id: number;
 
+  @Column({ type: 'jsonb', nullable: true })
+  @Transform(({ value }) => {
+    try {
+      return typeof value === 'string' ? JSON.parse(value) : value;
+    } catch {
+      return value;
+    }
+  })
+  detailedFeedback: Object;
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)